ledger/trie/leaf_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package trie_test
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger/common/utils"
+	"github.com/optakt/flow-dps/ledger/trie"
+)
+
+func Test_ProperLeaf(t *testing.T) {
+	path := utils.PathByUint16(56809)
+	payload := utils.LightPayload(56810, 59656)
+	n := trie.NewLeaf(0, path, payload)
+	expectedRootHashHex := "0ee164bc69981088186b5ceeb666e90e8e11bb15a1427aa56f47a484aedf73b4"
+	got := n.Hash()
+	require.Equal(t, expectedRootHashHex, hex.EncodeToString(got[:]))
+}
+
+func Test_CompactLeaf(t *testing.T) {
+	path := utils.PathByUint16(56809)
+	payload := utils.LightPayload(56810, 59656)
```

Comment:
Its not clear what is 56809, 56810 and 59656

Reply:
It is not clear in the original implementation either, but to be sure that we do not make breaking changes we need to have the same specification (tests) as they do. I think they also copied it from another implementation, in Python. I'll look into it to see if I can find the original one and get an explanation.

Reply:
Should be better now, I added a comment to it. LMK if it works for you :)